source/extensions/common/wasm/context.cc

```diff
@@ -1161,6 +1161,7 @@ WasmResult Context::log(uint64_t level, absl::string_view message) {
     ENVOY_LOG(critical, "wasm log{}: {}", log_prefix(), message);
     return WasmResult::Ok;
   case spdlog::level::off:
+  default:
```

Comment:
Not the core of your change, but `spdlog::level::off` should probably be a no-op and simply `return WasmResult::Ok`.

cc @jplevyak

Reply:
I agree.